index.html

```diff
@@ -220,6 +219,95 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstname"
+							id="firstname"
+							maxlength="30"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastname"
+							id="lastname"
+							maxlength="30"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
```

Comment:
- [ ] I can see that you are using two `input` fields for name. Great work implementing this 👍 However, please make sure that you stick to the requirements and use only one `input` field. Following is a quote for a related requirement 🔽

  > _IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions._